lib/view_model/wallet_address_list/wallet_address_edit_or_create_view_model.dart

```diff
@@ -105,7 +106,7 @@ abstract class WalletAddressEditOrCreateViewModelBase with Store {
         .setLabelSubaddress(
           wallet,
           accountIndex: monero!.getCurrentAccount(wallet).id,
-          addressIndex: _item.id as int,
+          addressIndex: _item?.id as int,
```

Comment:
`as int?` and provide a default value if applicable or check for item nullability first if required

Reply:
also please don't skip ci for this commit so we can verify the class type does not affect anything